clangify.sh

```diff
@@ -12,7 +16,33 @@ if hash clang-format 2>/dev/null; then
 		-not -path "./oracle/src/zip/*" \
 		-not -path "./build*/*" \
 		-exec clang-format -style=file -i {} \;
-	echo "Repository properly formatted"
+
+		if [[ $TRAVIS = true ]]; then
+			# if run from Travis CI, we want to compare the changes and report back to the contributor
+			git clean -f
+			if git diff --quiet; then
+				# exit code = 0 (no differences)
+				echo "Code style - OK"   ## 
+			else (
+				# exit code = 1 (differences)
+				echo "";
+				echo "*****************************************************";
+				echo "***  This PR is not clean against our code style  ***";
+				echo "***  Run clang-format and fix up any differences  ***";
+				echo "***  Check our CONTRIBUTING.md file for details!  ***";
+				echo "***                  Thank you ♥                  ***";
+				echo "*****************************************************";
+				)
+			fi
+			git diff --exit-code ## correct position? what is this used for? original script: https://github.com/Cockatrice/Cockatrice/blob/994a643d9c4e08dad1dfd7b13272598fdb42b196/.ci/travis-compile.sh
```

Comment:
At the beginning of this script there's a `set -e` that means (man page content):
```
 -e  Exit immediately if a command exits with a non-zero status.
```
Now, `git diff --exit-code` is used to: (from: https://git-scm.com/docs/git-diff#git-diff---exit-code)
```
Make the program exit with codes similar to diff(1). That is, it exits with 1 if there were differences and 0 means no differences.
```
Basically, the idea is that if clang-format makes any change to the files, `git --diff` will notice it and exit with a status of 1, marking the script (and thus the build) as failed.
This probably won't work anymore since this check is now inside the `clangify.sh` subscript, and nothing is checking its exit code.

Reply:
TODO